cloud/shared/bin/destroy

```diff
@@ -0,0 +1,13 @@
+#! /usr/bin/env bash
+
+set -e
+
+source cloud/shared/bin/lib.sh
+
+# This setups the environment of civiform
+# it will execute a bunch of scripts that help setup terraform, and the
+# various cloud providers.
+
+export TF_VAR_image_tag="${IMAGE_TAG}"
```

Comment:
do we need this?

Reply:
When I don't include this, the terraform template prompts on the commandline for a tag

Reply:
yeah that's cuz it's relying on the exported one existing in the config.sh but we actually moved away from including it there. i think in this case it just doesn't matter what the tag is so we should export like 'latest' or some string so we don't fail if you don't have the IMAGE_TAG in the env variables

Reply:
done